### ISSUE TITLE
Feature Request: Allow to pass "name" via command line switch

### ISSUE THREAD

**Issue by tec-lab** [tancredi/fantasticon#8]
The output file names seem fixed to _icons_.css, _icons_.eot, _icons_.woff, ...
Also the icons prefix is fixed to _icon_.

Please consider having a command line switch to configure the filename and CSS prefix.

thx

**Comment by tec-lab** [comment]
ok - the prefix can be changed using the --selector switch. What about the file names?
thx

**Comment by janbiasi** [comment]
I think it's only supported when using the config (`config.name`)

**Comment by tancredi** [comment]
Hi @tec-lab - so, the `name` option will define the base filename for all generated fonts and assets, and the actual font name generated in the CSS.

The option so far wasn't available from the command line since there was some namespace conflict issue with `commander`, but I've just addressed this in version `1.0.12` - and you can now see it in the docs.

Regarding changing the individual asset paths with more granularity, if you use a config file you'll be able to specify a filepath for each of them by using `pathOptions`, although some things may be a bit flimsy since so far the tool won't sort out very well relativities between the assets (if you care about the HTML preview file and you move the CSS, the HTML file may not point to the right stylesheet, but I guess it's pretty minor as the HTML is only relatively important).

Here's an example config file with custom paths:

```
module.exports = {
  // ..
  pathOptions: {
    ts: './src/types/icon-types.ts',
    json: './misc/icon-codepoints.json',
    css: './css/my-icon-font.css',
  }
};
```

Hope this helps, let me know if you have further questions
